src/cli/global/upgrade_all.rs

```diff
@@ -21,6 +21,10 @@ pub struct Args {
     /// The platform to install the package for.
     #[clap(long, default_value_t = Platform::current())]
     platform: Platform,
+
+    /// Do not insert conda_prefix, path modifications, and activation script into the installed executable script.
+    #[clap(long)]
+    no_activation: bool,
```

Comment:
Let's not add it to `upgrade-all` for now.

I think we should have a separate command that removes/adds activation for a specific binary.

Will think how to expose that here: https://github.com/prefix-dev/pixi/blob/main/docs/design_proposals/pixi_global_manifest.md